src/auspex/instruments/bbn.py

```diff
@@ -33,6 +33,22 @@
         aps2_missing = True
         aps2 = MagicMock()
 
+aps1_missing = False
+if config.auspex_dummy_mode:
+    fake_aps1 = True
+    aps1 = MagicMock()
+else:
+    try:
+        import aps as libaps
+        if libaps.APS_PY_WRAPPER_VERSION < 1.4:
+            logger.warning("Old version of libaps found. Using MagicMock instead.")
+            raise ImportError
+        fake_aps1 = False
+    except:
+        fake_aps1 = True
```

Comment:
Should we add a warning?

Reply:
Maybe like we do for the aps2:
https://github.com/BBN-Q/Auspex/blob/feature/aps-driver/src/auspex/instruments/bbn.py#L442